pkg/network/controllers/vm_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"context"
 	"errors"
 
+	libvmistatus "kubevirt.io/kubevirt/pkg/libvmi/status"
+
```

Comment:
Please add a test of a hotplugged NIC with link state `up` / `down`.
It could be easily added here [1].

[1] https://github.com/kubevirt/kubevirt/blob/9a8259e2bf354917fe5ef69672acb87dd2b5f412/pkg/network/controllers/vm_test.go#L132